cmd/server/assets/users/index.html

```diff
@@ -49,45 +61,105 @@
       </div>
 
       {{if $memberships}}
-        <table class="table table-bordered table-striped table-fixed table-inner-border-only border-top mb-0">
-          <thead>
-            <tr>
-              <th scope="col">Name</th>
-              <th scope="col" width="300">Email</th>
-              {{if $canWrite}}
+        <form method="POST" id="users-form">
```

Comment:
I think I'll need to try this out and see it in action to really verify this.

How are we going to document this? Should `#search` above have a `?` / `advanced` link with a pop up dialog explaining the advanced query options?

/lgtm
/hold

Reply:
I added an advanced query box